### PR TITLE
Calculation class

### PR DESCRIPTION
**Pull request by MisaOgura**

### CODE REVIEW

File: app/controllers/alarms_controller.rb [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+    @alarm = Alarm.last
+    calculate_time_to_leave
+    render json: @time_to_leave
   end
 
-  # GET /alarms/1
-  # GET /alarms/1.json
   def show
     render json: @alarm
+
```
Comment (shaneoston72) [inline app/controllers/alarms_controller.rb]: remove this line.

File: app/controllers/alarms_controller.rb [hunk 2]
```diff
@@ -36,15 +36,37 @@ def update
...
+    def format_arrival_time
+      @arrival_time = { hours: @alarm.arrival_time[0,2].to_i,
+                         minutes: @alarm.arrival_time[3,2].to_i}
+    end
+
+    def get_duration
+      from_station = @alarm.from_station.to_i
+      to_station = @alarm.to_station.to_i
+      travel = TravelApi.new(from_station, to_station).grab_json
+      p @duration = travel['journeys'][0]['duration']
```
Comment (shaneoston72) [inline app/controllers/alarms_controller.rb]: do we need this line? the p....

**Comment by shaneoston72** [comment]
:shipit:

**Comment by Managram** [comment]
:shipit:

**Pull request merged**